src/ui/component/fileActions/view.jsx

```diff
@@ -1,53 +1,25 @@
 // @flow
-import type { ElementRef } from 'react';
 import * as MODALS from 'constants/modal_types';
 import * as ICONS from 'constants/icons';
 import React from 'react';
 import Button from 'component/button';
 import Tooltip from 'component/common/tooltip';
-import { requestFullscreen, fullscreenElement } from 'util/full-screen';
-
-type FileInfo = {
-  claim_id: string,
-};
 
 type Props = {
   uri: string,
   claimId: string,
   openModal: (id: string, { uri: string }) => void,
   claimIsMine: boolean,
-  fileInfo: FileInfo,
-  viewerContainer: { current: ElementRef<any> },
-  showFullscreen: boolean,
+  fileInfo: FileListItem,
 };
 
 class FileActions extends React.PureComponent<Props> {
-  maximizeViewer = () => {
-    const { viewerContainer } = this.props;
-    const isFullscreen = fullscreenElement();
-    // Request fullscreen if viewer is ready
-    // And if there is no fullscreen element active
-    if (!isFullscreen && viewerContainer && viewerContainer.current !== null) {
-      requestFullscreen(viewerContainer.current);
-    }
-  };
-
   render() {
-    const { fileInfo, uri, openModal, claimIsMine, claimId, showFullscreen } = this.props;
-    const showDelete = claimIsMine || (fileInfo && Object.keys(fileInfo).length > 0);
+    const { fileInfo, uri, openModal, claimIsMine, claimId } = this.props;
+    const showDelete = claimIsMine || (fileInfo && (fileInfo.written_bytes > 0 || fileInfo.blobs_completed === true));
```

Comment:
Is `blobs_completed` a number or a boolean? this contrasts with https://github.com/lbryio/lbry-desktop/pull/2707/files#diff-90966bed4cd947a3f12fa8f891fd193aR35 below

Reply:
It is a number, good catch

Reply:
Blobs completed is a count